### PR TITLE
Enhance navbar modified style.css #132

### PR DESCRIPTION
**Pull request by Kritikaagra**

### CODE REVIEW

File: static/home/css/style.css [hunk 1]
```diff
@@ -19,9 +19,18 @@ header{
...
 .button{
-	border: 1px red solid;
-	border-radius: 10px;
 	cursor: pointer;
+	font-family:Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+	position: relative;
+	text-align: center;
+	color: black;
+	border: none;
+	transition: 0.4s ease;
```
Comment (tmo66) [inline static/home/css/style.css:28]: Don't change Global Variables.

File: static/home/css/style.css [hunk 2]
```diff
@@ -32,13 +41,19 @@ a{
 	padding: 0px 0px;
 	text-decoration: none;
 	color: black;
-	/*display: inline-block;*/
+	font-family:Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+	font-size: 18px;
 }
-a:hover{
+.navbar a:hover{
```
Comment (tmo66) [inline static/home/css/style.css:47]: Don't make changes here, do these changes in `navbar a: hover`.

File: static/home/css/style.css [hunk 3]
```diff
@@ -54,7 +69,7 @@ a:hover{
   ::-webkit-scrollbar-thumb:hover {
 	/* background: var(--theme-dark-color-0); */
 	/* background: #666; */
-  }
+  }
```
Comment (tmo66) [inline static/home/css/style.css:72]: Don't add extra whitespace and an extra line.

File: static/home/css/style.css [hunk 4]
```diff
@@ -101,17 +116,21 @@ nav{
 	top: 0;
 	background-color: #ff8c00;
 	z-index: 2;
+	font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;	
 }
+
```
Comment (tmo66) [inline static/home/css/style.css:121]: Don't add an extra line here.

File: static/home/css/style.css [hunk 5]
```diff
@@ -101,17 +116,21 @@ nav{
...
 .navbut{
-	border: 1px red solid;
+	border: 1px black solid;
 	border-radius: 4px;
 	background-color: #ff8c00;
 	padding: 3px 10px;
 	cursor: pointer;
 	transition: 0.5s;
+	margin: 5.5px;
+
```
Comment (tmo66) [inline static/home/css/style.css:130]: Don't add an extra line here.

File: static/home/css/style.css [hunk 6]
```diff
@@ -101,17 +116,21 @@ nav{
...
 	background-color: #ff8c00;
 	padding: 3px 10px;
 	cursor: pointer;
 	transition: 0.5s;
+	margin: 5.5px;
+	
 }
 .navbut:hover{
-	background-color: red;
+	background-color:red;
```
Comment (tmo66) [inline static/home/css/style.css:133]: Don't remove the spacing between both. It's how the code is formatted throughout the project.

File: static/home/css/style.css [hunk 7]
```diff
@@ -182,7 +204,7 @@ nav{
 /*navbar in full size*/
 .navbar{
 	float: left;
-	/*margin: 5px 0;*/
```
Comment (tmo66) [inline static/home/css/style.css:185]: Don't remove this.

File: static/home/css/style.css [hunk 8]
```diff
@@ -191,28 +213,25 @@ nav{
 }
 .navbar li{
 	display: inline;
-	/*padding: 0 5px;*/
```
Comment (tmo66) [inline static/home/css/style.css:194]: Don't remove this.

File: static/home/css/style.css [hunk 9]
```diff
@@ -191,28 +213,25 @@ nav{
...
 	position: absolute;
-	top: 32px;
+	top: 45px;
 	left: 0px;
 	padding: 0;
 	width: 100%;
 	background-color: rgba(255,140,0,0.9);
-	box-sizing: border-box;
-	height:0px;
+	/*box-sizing: border-box;*/
```
Comment (tmo66) [inline static/home/css/style.css:234]: Don't delete anything which is not created by you. Just put them in the comment. And delete all those which are written by you but later on commented.

**Review by tmo66 — CHANGES_REQUESTED** [review]
Make all these necessary changes.

File: static/home/css/style.css [hunk 10]
```diff
@@ -136,21 +155,23 @@ nav{
...
 	display: inline;
-	padding: 0 5px;
+	padding: 0px 0px;
 	color: black;
 }
 .navbar1 .button{
 	position: relative;
-	padding: 3px 10px;
+	/*margin: 3px 5px;*/
+	padding: 13.5px 11px;
```
Comment (tmo66) [inline static/home/css/style.css:164]: Not sure about 13.5 px...it's too much!

**Comment by Kritikaagra** [comment]
done with all necessary changes...
set all global variable as their previous values
remove all extra lines and comments added by me
also added lines I deleted ... and make them as comment to make changes

If I decrease padding from 13.5, the red colour on hover does not cover the whole space and leave a line at bottom.

File: static/home/css/style.css [hunk 11]
```diff
@@ -136,21 +139,25 @@ nav{
 }
 .navbar1 li{
 	display: inline;
-	padding: 0 5px;
+	padding: 0px 0px;
```
Comment (tmo66) [inline static/home/css/style.css:142]: No need to write this..

File: static/home/css/style.css [hunk 12]
```diff
@@ -136,21 +139,25 @@ nav{
...
+	font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+	border: none;
+	border-radius: 0px;
 	overflow: hidden;
 	transition: 0.5s ease;
 }
 .navbar1 .button:hover{
-	background-color: red;
 	color: white;
-	padding: 3px 12px;
```
Comment (tmo66) [inline static/home/css/style.css:153]: Don't remove these two-line, just put them in comment.

File: static/home/css/style.css [hunk 13]
```diff
@@ -159,9 +166,10 @@ nav{
 	left: 0;
 	right: 0;
 	bottom: 0;
-	width: 1px;
+	width: 100%;
+	height: 100%;
 	margin: auto;
-	background: red;
+	background: white;
```
Comment (tmo66) [inline static/home/css/style.css:172]: No need to make changes here, as this whole part is already commented out.

File: static/home/css/style.css [hunk 14]
```diff
@@ -234,6 +246,9 @@ nav{
 .navbar2 li a{
 	display: block;
 }
+.navbar2 a:hover{
```
Comment (tmo66) [inline static/home/css/style.css:249]: Don't make these changes, for now, we are handling navbar2 in a different issue.

**Review by tmo66 — CHANGES_REQUESTED** [review]
Just make these small changes, everything else looks good to me! And also after the commit squash all the commits in one.

**Comment by tmo66** [comment]
To squash your commits, follow these instructions:
* `git rebase -i HEAD~<no. of commits you made in total>`
* Except for first `pick`, replace all `pick` with `squash` in the file opened.
* Close the file after saving it.
* In the new file opened, remove all commit messages and write a new one and then close this file too after saving.
* Force push your changes. (`git push origin +<your-branch-name>`).

**Comment by garg3133** [comment]
Thank you @Kritikaagra for working on this. The enhancements were good but the navbar was not being displayed properly on small screens due to which I was reluctant to merge this PR. But then I thought, why not rebuild the entire navbar from scratch as it was very badly implemented (you must have seen there were two navbars, one for small screens and the other of larger screens). So, I spent some time on it and re-built it.

But, as your enhancements were good and I've taken some idea from this PR itself while rebuilding it, I'm still merging this PR so that you get your points for GSSoC. And I'm really sorry you didn't get your changes in the master branch. Hope you'll understand and continue making your amazing contributions. Thanks!

**Comment by garg3133** [comment]
Also, you should never make any changes directly into the master branch.

**Comment by Kritikaagra** [comment]
Thank you @garg3133
And sorry for making changes in the master branch. I will take care this next time.

**Pull request merged**